lib/config.coffee

```diff
@@ -67,4 +67,10 @@ module.exports = {
     default: false
     description: "Mute 'Latest backup is already applied' message"
     order: 14
+  removePackage:
+    title: 'Remove unused packages'
```

Comment:
Can you please update this to `Remove obsolete packages`. (The following description is good as it is I think.)